docs/dev/new_check_howto.md

```diff
@@ -6,77 +6,126 @@ aliases:
-
-There are several options for installing Python, including:
-
-- Following the [official Python documentation][3] to download and install the Python interpreter.
-- Using a Python version manager like [pyenv][4].
+- Python v3.8 and [pipx][2]. For instructions on installing Python and pipx, see [Python for Agent Integration Development][3].
+- [Docker][4] to run the full test suite
+- The git [command-line][5] or [GitHub desktop client][19]
 
-Python v3.3 or later comes with a pre-installed version manager called `venv`, which is used on this page. Debian or Ubuntu installations do not come packaged with `venv`. You can install the `venv` package by running `sudo apt-get install python3-venv`.
+## Set up your integrations-extra repo
 
-## Prepare your development environment
+Follow these instructions to set up your repo for integration development:
 
-Follow these instructions to set up your development environment:
-
-1. Create the `dd` directory and clone the [`integrations-extras` repo][5].
+1. Create the `dd` directory:
 
    The Datadog Development Toolkit expects you to work in the `$HOME/dd/` directory. This is not mandatory, but working in a different directory requires additional configuration steps.
 
    To create the `dd` directory and clone the `integrations-extras` repo:
-   ```shell
+   ```
    mkdir $HOME/dd && cd $HOME/dd
-   git clone https://github.com/DataDog/integrations-extras.git
    ```
 
-1. Optionally, set up a [Python virtual environment][6] to isolate your development environment:
+1. Fork the the [`integrations-extras` repo][6].
+
+1. Clone your fork into the `dd` directory:
+   ```
+   git clone git@github.com:<YOUR USERNAME>/integrations-extras.git
+   ```
 
-   ```shell
-   cd $HOME/dd/integrations-extras
-   python3 -m venv venv
-   . venv/bin/activate
+1. Create a feature branch to work in:
+   ```
+   git switch -c <YOUR INTEGRATION NAME> origin/master
    ```
 
-   **Tip**: If you ever need to exit the virtual environment, run `deactivate`.
+## Install the Development Toolkit
 
-1. Ensure the Python `wheel` package is installed and up-to-date:
+Assuming you've [installed Python and pipx][3], install the Development Toolkit for your operating system:
 
-   ```shell
-   pip3 install wheel
+{{< tabs >}}
+{{% tab "MacOS" %}}
+
+1. Run the following command and remove any executables shown in the output:
+   ```
+   which -a ddev
    ```
 
-1. Install the [Developer Toolkit][7]:
+1. Make sure there are no virtual environments running:
+   1. Run the following command:
+      ```
+      echo VIRTUAL_ENV
+      ```
+
+   1. If the command returns output, a virtual environment is running. Run `deactivate` to exit the virtual environment.
+
+1. Install `ddev`:
+   <div class="alert alert-warning">Do not run this command with <code>sudo</code>.</a></div>
 
-   ```bash
-   pip3 install "datadog-checks-dev[cli]"
+   ```
+   pipx install ddev --python /usr/local/opt/python@3.8/bin/python3.8
```

Comment:
This path wasn't available for me. I had to run:
> pipx install ddev --python /opt/homebrew/opt/python@3.8/libexec/bin/python3

Reply:
Thanks, I think this might be an M1 vs Intel Mac thing.

Reply:
Oh, another thing to note is that `pipx` output included a command I had to run to put the newly installed `ddev` in my path. Since we included wording about `homebrew` above that did this, it'd probably be helpful to mention it here too. Specifically:
> These apps will not be globally accessible until your PATH is updated. Run `pipx ensurepath` to automatically add it, or manually modify your PATH in your shell's config file